be/src/formats/parquet/metadata.cpp

```diff
@@ -482,8 +482,7 @@ StatusOr<FileMetaDataPtr> FileMetaDataParser::get_file_metadata() {
         auto deleter = [](const CacheKey& key, void* value) { delete (FileMetaDataPtr*)value; };
         ObjectCacheWriteOptions options;
         options.evict_probability = _datacache_options->datacache_evict_probability;
-        st = _cache->insert(metacache_key, capture, file_metadata_size, file_metadata_size, deleter, &cache_handle,
-                            &options);
+        st = _cache->insert(metacache_key, capture, file_metadata_size, deleter, &cache_handle, &options);
```

Comment:
Is it necessary to accumulate the `sizeof(capture)`  and `file_metadata_size` to make the charge_size more accurate?

Reply:
I will optimize the detail later, currenty it's not necessary